apps/site/playwright.config.ts

```diff
@@ -10,11 +11,34 @@ export default defineConfig({
   retries: isCI ? 2 : 0,
   workers: isCI ? 1 : undefined,
   reporter: isCI ? [['html'], ['github']] : [['html']],
-  use: {
-    baseURL: process.env.VERCEL_PREVIEW_URL || 'http://127.0.0.1:3000',
-    trace: 'on-first-retry',
-  },
+  ...(() => {
+    const use: PlaywrightTestConfig['use'] = {
+      baseURL: process.env.PLAYWRIGHT_BASE_URL || 'http://127.0.0.1:3000',
+      trace: 'on-first-retry',
+    };
 
+    const webServerCommand = process.env.PLAYWRIGHT_WEB_SERVER_COMMAND;
+    const webServerPort = parseInt(
+      process.env.PLAYWRIGHT_WEB_SERVER_PORT ?? ''
+    );
```

Comment:
Can we simplify this to just

`PLAYWRIGHT_WEB_SERVER_COMMAND` and `PLAYWRIGHT_BASE_URL`, no need for `PLAYWRIGHT_WEB_SERVER_PORT`, right?

Reply:
good call, updated! 🙂